enterprise/backend/src/metabase_enterprise/transforms/interface.clj

```diff
@@ -0,0 +1,31 @@
+(ns metabase-enterprise.transforms.interface)
+
+(defn- transform->transform-type
+  [transform]
+  (-> transform :source :type keyword))
+
+(defmulti source-db-id
+  "Return the source db id for a given transform."
```

Comment:
Non blocking, we can change it later - but what about pythonic cross database transforms? I know its not exposed by IIRC the code is still set up to support it (we are paying that cost?)